doc/installation.rst

```diff
@@ -243,7 +243,7 @@ Make sure you have Xcode installed, including its command-line tools
 If you are using `Homebrew <https://brew.sh>`_::
 
     # brew install bison flex cmake ninja python@3.8 sphinx-doc
-    # pip3 install btest sphinx_rtd_theme
+    # pip3 install btest sphinx_rtd_theme diagrams
```

Comment:
Nit: Since I have seen people repeatedly destroy their (in particular macos) Python installation with `pip install` outside of venvs I wonder whether we shouldn't just not mention doc dependencies at all here and move from `pip3 install` to `pipx install` to suggest a managed venv for btest ([pipx](https://pipx.pypa.io/stable/) only manages executables).

Reply:
Let me split it out at least.